docs/root/configuration/statistics.rst

```diff
@@ -0,0 +1,45 @@
+.. _statistics:
+
+Statistics
+==========
+
+A few statistics are emitted to report statistics system behavior:
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: 1, 1, 2
+
+  stats.overflow, Counter, Total number of times Envoy cannot allocate a statistic due to a shortage of shared memory
+
+Server
+------
+
+Server related statistics are rooted at *server.* with following statistics:
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: 1, 1, 2
+
+  uptime, Gauge, Current server uptime in seconds
+  memory_allocated, Gauge, Current amount of allocated memory
+  memory_heap_size, Gauge, Current reserved heap size
+  live, Gauge, "*1* if the server is not currently draining, *0* otherwise"
```

Comment:
Sorry you can remove the quotation marks to match other entries. That was just me quoting what to write. :)

Reply:
OK, just removed `*` chars. 9454fa4

But did you mention to the double quotation mark (`"`)? If so, it is for csv-table syntax, and it does not affect generated words:

![image](https://user-images.githubusercontent.com/1460043/37611592-156933c0-2be6-11e8-96ca-24ceab168071.png)

Reply:
We need to quote to include a comma (`,`) into the sentence.

Reply:
Oops sorry, nevermind. I thought the quotes would show up.

Reply:
No problem, thanks for reviewing!